src/rpc/connectivity/cluster.cc

```diff
@@ -400,6 +412,7 @@ void connectivity_cluster_t::run_t::join_blocking(
 
     // Attempt to connect to all known ip addresses of the peer
     bool successful_join = false; // Variable so that handle() can check that only one connection succeeds
+    join_result_t join_result; // Used to determine the joint result for an individual connection attempt
```

Comment:
Is the `joint` a typo or should the comment say "the joint join result across all individual connection attempts"?